plans/tier1.fmf

```diff
@@ -242,6 +242,9 @@
           playbook: tests/ansible_collections/roles/reboot/main.yml
 
 /system_up_to_date:
+    # There are no minor version repositories for Oracle Linux EUS releases, so we skip the check for up-to-date system.
```

Comment:
```suggestion
    # There are no repositories for older minor versions of Oracle Linux, so we skip the check for up-to-date system.
```
There's nothing like OL EUS release.

Reply:
Is the message relevant at all at this point? Since we do not test on older OL versions, there is nothing to skip, right? @danmyway

Reply:
You're right, I will remove the comment.